### ISSUE TITLE
focus on video

### ISSUE THREAD

**Issue by SajadRahimi1** [media-kit/media-kit#496]
Hi
I want to focus on Video by Dpad. Indeed I want to show Material Buttons by click on Dpad.
Is There any solution to it?

**Comment by LNA-DEV** [comment]
Are there any news on this topic?

**Comment by LNA-DEV** [comment]
If I understand this correctly there is currently no way to make the (default) video overlay work on Android TV? It seems like the focus and shortcuts get in each others way? Or am I missing something? @alexmercerind

**Comment by alexmercerind** [comment]
The video/audio playback part is the focus of this libr

**Comment by andynvt** [comment]
I need this too. @LNA-DEV Do you have PR yet?

**Comment by andynvt** [comment]
@alexmercerind is there anyway to show/hide the control programmatically?

**Comment by LNA-DEV** [comment]
> I need this too. @LNA-DEV Do you have PR yet?

@andynvt I am not working actively on a PR. But I have this code for you: https://github.com/OpenMediaStation/OpenMediaStation.FE.MovieTV/blob/620bfd937c27b4619f709c4edab26de08888ed27/lib/player_controls/material_tv.dart

This should only be used on devices without touch but for TV it is working relatively good. Either way you can see how it works.

@alexmercerind Is there an interest implementing something like this into the package?

**Comment by alexmercerind** [comment]
> @alexmercerind is there anyway to show/hide the control programmatically?

> @alexmercerind Is there an interest implementing something like this into the package?

No. Why am I obliged?

**Comment by alexmercerind** [comment]
I think you can create a package for your controls, we can list it in our README.

**Comment by LNA-DEV** [comment]
I do not think it is a big enough change to make a own package for the control. Maybe if it is a bit further developed.

**Comment by alexmercerind** [comment]
> I do not think it is a big enough change to make a own package for the control. Maybe if it is a bit further developed.

We can list them at https://github.com/media-kit/media-kit?tab=readme-ov-file#video-controls if you do publish.

To be frank, it's getting bit hard for me to take care of everything. I much rather focus on the core functionality of project.

**Comment by LNA-DEV** [comment]
I totally get that. I think I will just put a small package together and mention you if that is done.

**Comment by SP4RKiOP** [comment]
@LNA-DEV was just completing my app's D-PAD support and came to this issue page as i am using media_kit for video. Can you tell me what files i need to copy from your package to my project so that i can use the media_kit package but your Tv controls as optional for any non touchscreen device(TV). Thanks for your efforts

**Comment by SP4RKiOP** [comment]
@LNA-DEV have you explored how we can travel the topBar buttons?

**Comment by LNA-DEV** [comment]
@SP4RKiOP I do not know what you mean. On my project I can navigate the top bar with dpad. 

But for further discussions about the extension package I would suggest opening an issue there.

**Comment by SP4RKiOP** [comment]
> @SP4RKiOP I do not know what you mean. On my project I can navigate the top bar with dpad.
> 
> But for further discussions about the extension package I would suggest opening an issue there.

i already checked your implementation, in fact i copied and tested on my app replacing my player with yours (the whole code) but unfortunately i was not able to navigate the top bar button with dpad. 

btw i have created a issue on the extension package. and started implementing own code for issue.
